### ISSUE TITLE
Get's stuck on bigger files. Under both Windows and Linux

### ISSUE THREAD

**Issue by THPubs** [CraigWilliams/BeautifyRuby#82]
When I try to beautify large files (specially html.erb files), Sublime text gets stuck and I have to forcefully exit the application. Why is that?

**Comment by goooseman** [comment]
https://github.com/threedaymonk/htmlbeautifier/issues/17
closed

**Comment by THPubs** [comment]
Yes htmlbeautifier gem fixed the issue. But if we update to the fixed version of the gem, BeautifyRuby starts to throw this error : Error: invalid output. Check your ruby interpreter settings

**Comment by goooseman** [comment]
@THPubs for me everything works ok. maybe the problem in you $PATH

**Comment by THPubs** [comment]
@goooseman Hi, what is the path you use? Here's what I have used : 
    "ruby": "~/.rvm/bin/rvm-auto-ruby"

**Comment by goooseman** [comment]
I'm os OS X with RVM and I install all Sublime Text necessary gems to system ruby so I don't change plugin's path setting. I use
`rvm use system`
`sudo gem install htmlbeautifier`

**Comment by THPubs** [comment]
Im in Ubuntu

**Comment by goooseman** [comment]
@THPubs I don't know if there a default ruby shipped with Ubuntu.If there any, this code will work.
If not, I prefer just typing `echo $PATH` to terminal and copy-paste this string to settings.

**Comment by THPubs** [comment]
@goooseman After doing that the path will look like this `"ruby": "/home/pubudu/.rvm/gems/ruby-2.2.1/bin:/home/pubudu/.rvm/gems/ruby-2.2.1@global/bin:/home/pubudu/.rvm/rubies/ruby-2.2.1/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/home/pubudu/.rvm/bin:/home/pubudu/.rvm/bin"`

Now, not only for html.erb but also for only ruby files I get the error `Error: invalid output. Check your ruby interpreter settings`

**Comment by goooseman** [comment]
@THPubs because of this strange behaviors  I decided to use system ruby with RVM. With rbenv everything works well with `echo $PATH`. So i can't help you with this problem.
